### PR TITLE
Fs 880 tasklist

### PR DESCRIPTION
**Pull request by Adam-W1**

### CODE REVIEW

File: app/default/data.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import json
+import os
+
+import requests
+from app.config import FLASK_ROOT
+from app.config import GET_APPLICATION_ENDPOINT
+
+
+def get_data(endpoint: str):
+    if endpoint[:8] == "https://":
```
Comment (SentientHousePlant) [inline app/default/data.py:10]: 🤔 Try using [startswith](https://www.programiz.com/python-programming/methods/string/startswith) for better comprehension.

File: app/default/data.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        data = get_remote_data(endpoint)
+    else:
+        data = get_local_data(endpoint)
+    return data
+
+
+def get_remote_data(endpoint):
```
Comment (SentientHousePlant) [inline app/default/data.py:17]: 📌 This is a great candidate for monkey patching!

File: app/models/continue_application.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+    Args:
+        application_id (str): The id of an application in the application store
+        form_name (str): The name of the application sub form/section
+        page_name (str): The form page to redirect the user to.
+
+    Returns:
+        A function: given a users application id they are redirected to
+        the specified application form page within the form runner service
+    """
```
Comment (SentientHousePlant) [inline app/models/continue_application.py:24]: 💯 These docstrings slay.

File: app/models/tasklist.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+            )
+        ),
+        "number_of_incomplete_sections": len(
+            list(
+                filter(
+                    lambda section: section["status"] == "NOT_STARTED",
+                    application_response["sections"],
+                )
+            )
+        ),
```
Comment (SentientHousePlant) [inline app/models/tasklist.py:37]: 🔧 I believe that you can do this with a list comprehension...<3

**Review by SentientHousePlant — CHANGES_REQUESTED** [review]
Just some minor comments for you Adam, overall very nice.

File: app/default/data.py (hunk 1 above)
Comment (Adam-W1) [reply app/default/data.py:10]: oo i do love a new python method

File: app/models/tasklist.py (hunk 4 above)
Comment (Adam-W1) [reply app/models/tasklist.py:37]: can we pair up on this tomorrow?

**Pull request merged**